sklearn/tree/_utils.pyx

```diff
@@ -611,9 +597,9 @@ cdef class WeightedMedianCalculator:
                                                   original_median)
         return return_value
 
-    cdef int update_median_parameters_post_remove(self, DOUBLE_t data,
-                                                  DOUBLE_t weight,
-                                                  double original_median) nogil:
```

Comment:
I think I did this because `nogil` was hanging out of the 79 char limit... But yes sorry. I added `except *` and later removed it... Otherwise I wouldn't have touched this line I guess...